docs/source/reference/kubernetes/multi-kubernetes.rst

```diff
@@ -0,0 +1,146 @@
+        certificate-authority-data:
+        ...
+        server: https://yy.yy.yy.yy:45819
+      name: my-tpu-cluster
+    contexts:
+    - context:
+        cluster: my-h100-cluster
+        user: my-h100-cluster
+      name: my-h100-cluster
+    - context:
+        cluster: my-tpu-cluster
+        namespace: my-namespace
+        user: my-tpu-cluster
+      name: my-tpu-cluster
+    current-context: my-h100-cluster
+    ...
+
+
+In this example, we have two Kubernetes clusters: ``my-h100-cluster`` and ``my-tpu-cluster``, and each Kubernetes cluster has a context for it.
+
+Step 2: Setup SkyPilot to Access Multiple Kubernetes Clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unlike clouds, SkyPilot does not failover through different Kubernetes clusters (regions) by default because each Kubernetes clusters can have a different purpose.
+
+By default, SkyPilot only uses the current context in the kubeconfig, e.g., ``current-context: my-h100-cluster`` or you can get the current context with ``kubectl config current-context``.
+
+To allow SkyPilot to access multiple Kubernetes clusters, you can set the ``kubernetes.allowed_contexts`` in the SkyPilot config.
+
+.. code-block:: yaml
+
+    kubernetes:
+      allowed_contexts:
+        - my-h100-cluster
+        - my-tpu-cluster
+
+To check the enabled Kubernetes clusters, you can run ``sky check kubernetes``.
+
+.. code-block:: console
+
+    $ sky check kubernetes
+
+    🎉 Enabled clouds 🎉
+      ✔ Kubernetes
+        Allowed contexts:
+        ├── my-h100-cluster
+        └── my-tpu-cluster
+
+
+Failover across Multiple Kubernetes Clusters
+--------------------------------------------
+
+With the ``kubernetes.allowed_contexts`` global config, SkyPilot failover through the Kubernetes clusters in the ``allowed_contexts`` in the same
+order as they are specified.
+
+
+.. code-block:: console
+
+    $ sky launch --gpus H100 --cloud kubernetes echo 'Hello World'
+
+    Considered resources (1 node):
+    ------------------------------------------------------------------------------------------------------------
+    CLOUD        INSTANCE           vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE           COST ($)   CHOSEN   
+    ------------------------------------------------------------------------------------------------------------
+    Kubernetes   2CPU--8GB--1H100   2       8         H100:1         my-h100-cluster-gke   0.00          ✔     
+    Kubernetes   2CPU--8GB--1H100   2       8         H100:1         my-h100-cluster-eks   0.00                
+    ------------------------------------------------------------------------------------------------------------
+
+
+Point to a Kubernetes Cluster and Launch
+-----------------------------------------
+
+SkyPilot borrows the ``region`` concept from clouds to denote a Kubernetes context. You can point to a Kubernetes cluster
+by specifying the ``--region`` with the context name for that cluster.
+
+.. code-block:: console
```

Comment:
Hmm `console` renders the comments in regular colors:
<img width="863" alt="image" src="https://github.com/user-attachments/assets/f09737a5-7a56-42ef-a8c8-b9063fe09d80" />


maybe switch to `bash`: 
```suggestion
.. code-block:: bash
```